src/kirby/components/list/list.component.ts

```diff
@@ -172,4 +220,30 @@ export class ListComponent implements OnChanges {
     });
     return orderMap;
   }
+
+  private getItemOrder(item: any): { isFirst: boolean; isLast: boolean } {
+    const defaultOrder = { isFirst: false, isLast: false };
+    if (!item) {
+      return defaultOrder;
+    }
+    if (!this.isSectionsEnabled) {
+      return defaultOrder;
+    }
+    const order = this.orderMap.get(item);
+    if (!order) {
+      console.warn('Order of list item within section not found!');
+      return defaultOrder;
+    }
+    return order;
+  }
+
+  private initialzeSwipeActions(): void {
+    const large = style.global['$breakpoints'].value['large'].value;
```

Comment:
Consider leaving a `TODO` comment here, I believe we can find a better way to get the breakpoint...